package.json

```diff
@@ -15,14 +15,14 @@
     "start:cosmos": "nx run cowswap-frontend:cosmos:run",
     "build": "yarn run build:cowswap",
     "build:all": "nx run-many -t build",
-    "build:cowswap": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx build cowswap-frontend",
+    "build:cowswap": "cross-env nx build cowswap-frontend",
     "build:widget": "nx build widget-configurator",
-    "build:cosmos": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx run cowswap-frontend:build:cosmos",
+    "build:cosmos": "cross-env nx run cowswap-frontend:build:cosmos",
     "ci:build": "yarn run build:cowswap && yarn run build-widget",
     "ipfs:build": "PUBLIC_URL=\".\" yarn build",
     "ipfs:publish": "npx ipfs-deploy build -p pinata -O",
     "i18n": "nx run-many -t i18n",
-    "preview": "cross-env NODE_OPTIONS=--max-old-space-size=32768 nx run cowswap-frontend:preview",
+    "preview": "cross-env nx run cowswap-frontend:preview",
```

Comment:
If there are no env vars set, there's also no need for `cross-env`

Reply:
Fixed